build.rs

```diff
@@ -1,5 +1,25 @@
 use std::io::Write;
 
+const LINEAR_TO_SRGB_LOOKUP_SIZE: usize = 8192;
+
+fn linear_to_srgb(value: f32) -> u8 {
```

Comment:
sad that floating point arithmetic is not allowed in constant functions 🤓

Reply:
Oh well, not like the build script is gonna be faster that way :'-)